variables.subscription.tf

```diff
@@ -197,3 +197,55 @@ variable "wait_for_subscription_before_subscription_operations" {
 The duration to wait after vending a subscription before performing subscription operations.
 DESCRIPTION
 }
+
+variable "subscription_budgets" {
+  type = map(object({
+    amount            = number
+    time_grain        = optional(string, "Monthly")
+    time_period_start = string
+    time_period_end   = string
+    notifications = map(object({
+      enabled       = bool
+      operator      = string # EqualTo, GreaterThan, GreaterThanOrEqualTo
+      threshold     = number # 0-1000 percent
+      thresholdType = string # Actual, Forecasted
```

Comment:
As noted in other comments these could and should be snake case.

Reply:
added local to transform the notification map.